spec/spec_helper.rb

```diff
@@ -1,3 +1,5 @@
+require 'coveralls'
+require 'support/login_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.